test/SchemaUtils.test.ts

```diff
@@ -322,6 +322,9 @@ describe('SchemaUtils', () => {
               selectors: [['aria/Test']],
               operator: '==',
               count: 1,
+              properties: {},
```

Comment:
let's add tests with non empty attributes too. And a test when properties or attributes are of invalid types,

Reply:
What a help